footer.php

```diff
@@ -119,17 +119,17 @@
 <div class="credits section bg-dark small-padding">
 	<div class="credits-inner section-inner">
 <div class="logos">
-	<a href="https://floss.social/@fedoracz" rel="publisher"><i class="fab fa-mastodon" aria-hidden="true"></i></a>
-	<a href="http://www.facebook.com/fedoracz"><i class="fa fa-facebook" aria-hidden="true"></i></a>
-	<a href="http://twitter.com/fedoracz"><i class="fa fa-twitter" aria-hidden="true"></i></a>
-	<a href="https://t.me/mojefedora"><i class="fa fa-paper-plane" aria-hidden="true"></i></a>
-	<a href="https://www.youtube.com/channel/UCLCwHfGfcbDfs9E8lldJPQw"><i class="fa fa-youtube" aria-hidden="true"></i></a>
+	<a target="_blank" href="https://floss.social/@fedoracz" rel="me"><i class="fab fa-mastodon" aria-hidden="true"></i></a>
+	<a target="_blank" href="https://www.facebook.com/fedoracz"><i class="fa fa-facebook" aria-hidden="true"></i></a>
+	<a target="_blank" href="https://twitter.com/fedoracz"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+	<a target="_blank" href="https://t.me/mojefedora"><i class="fa fa-paper-plane" aria-hidden="true"></i></a>
+	<a target="_blank" href="https://www.youtube.com/channel/UCLCwHfGfcbDfs9E8lldJPQw"><i class="fa fa-youtube" aria-hidden="true"></i></a>
```

Comment:
Odkaz na YouTube je mrtvý. Je nějaká náhrada, nebo ho spíš smazat?

Reply:
Smazl bych to, nenapada me adekvatni nahrada.